Formula/microws.rb

```diff
@@ -0,0 +1,42 @@
+class Microws < Formula
+  desc "Tiny WebSockets"
+  homepage "https://github.com/uNetworking/uWebSockets"
+  url "https://github.com/uNetworking/uWebSockets/archive/v0.14.4.tar.gz"
+  sha256 "9d7854fe3b8c43903957ccd9001e954e210e14a0545549515a9f020bf275c4fc"
+
+  depends_on "libuv"
+  depends_on "openssl"
+
+  def install
+    system "make", "Darwin"
+    (include/"uWS").install Dir["src/*.h"]
+    lib.install "libuWS.dylib"
```

Comment:
We'll need upstream to fix their "make install" target (https://github.com/uNetworking/uWebSockets/blob/master/Makefile#L20-L23) so that it can handle
```
make install PREFIX=/usr/local/Cellar/microws/0.14.4
```
rather than hard coding its prefix.

Reply:
Request for upstream to modify the Makefile is [here](https://github.com/uNetworking/uWebSockets/pull/751).

Reply:
Upstream has merged change, but changes in Makefile will not be available until next release, so this formula will have to be on hold until then.

Reply:
I can make a release if you want to

Reply:
If you think it's a big enough change to warrant it, feel free to. I personally have no rush to get this merged, so it can wait if there haven't been many changes since the last release.

Reply:
Hahaaa, I've already released it.